### DISCUSSION TITLE
itchat如何实现桌面微信登录

### DISCUSSION THREAD

**Discussion by CornerLittleDust** [zhayujie/chatgpt-on-wechat#1307] [Q&A]
我导入了itchat官网的itchat库，看网上说enableCmdQR设为2就可以实现桌面登录，像下面这样
```python
import itchat

itchat.auto_login(enableCmdQR=2)
``` 
但是我调用itchat官网的auto_login方法却一直是网页版登录，而调用作者的lib.itchat却可以实现桌面登录。想问下作者还有大家具体是lib.itchat下的哪个方法哪行代码发挥了作用，或者提供点能够实现微信桌面登录的思路。

**Comment by BaronDJ** [comment]
我想用lib.itchat，请求作者给解答一下。
